development/image-syncer/README.md

```diff
@@ -7,8 +7,10 @@ The main use case is to preserve images from third party registries in our own r
 
 Syncing process steps:
 1. Pull image from source.
-2. Check if image exists in target.
-3. If image does not exist: re-tag image and push to target registry.  
+2. Check if image name contains SHA256 digest.
+3. If image name contains digest: re-tag target image with the tag instead of SHA256 digest.
+4. Check if image exists in target.
+5. If image does not exist: re-tag image and push to target registry.  
```

Comment:
```suggestion
3. If image name contains digest, re-tag target image with the tag instead of SHA256 digest.
4. Check if image exists in target.
5. If image does not exist, re-tag image and push to target registry.  
```